cluster/pulumi/cluster/src/fluentBit.ts

```diff
@@ -0,0 +1,122 @@
+      "    DB                /var/log/flb_kube.db",
+      "    Mem_Buf_Limit     100MB",
+      "    Skip_Long_Lines   On",
+      "    Refresh_Interval  1",
+      "    Buffer_Max_Size   20MB",
+      "    Read_from_Head True",
+      "    Tag kube.*"
+    ].join("\n"),
+    outputs: [
+      // Stack driver
+      "[Output]",
+      "    Name stackdriver",
+      "    Match kube.*",
+      // This ensures that non-JSON fields get parsed as textPayload
+      "    text_payload_key message",
+      // This ensures that the k8s labels added by the filter get propagated to the labels field in log explorer
+      "    labels_key k8s_labels",
+      "    resource k8s_container",
+      // Mandatory parameters, doesn't look like we have a way to exclude the location sadly even though it isn't very useful for us.
+      `    k8s_cluster_name ${CLUSTER_NAME}`,
+      `    k8s_cluster_location  ${GCP_REGION}`,
+    ].join("\n"),
+    filters: [
+      // First parse just in containerd format to extra time stamps and friends
+      "[FILTER]",
+      "    Name         parser",
+      "    Match        kube.*",
+      "    Key_Name     log",
+      "    Reserve_Data True",
+      "    Parser       containerd",
+      // Rename log to message as this is what stack driver expects
+      "[FILTER]",
+      "    Name        modify",
+      "    Match       *",
+      "    Hard_rename log message",
+      // Try to parse as glog and json, that matches what the default gc configuration does"
+      "[FILTER]",
+      "    Name         parser",
+      "    Match        kube.*",
+      "    Key_Name     message",
+      "    Reserve_Data True",
+      "    Parser       glog",
+      "    Parser       json",
+      // Enrich with k8s metadata
+      "[FILTER]",
+      "    Name             kubernetes",
+      "    Match            kube.*",
+      "    Kube_URL         https://kubernetes.default.svc:443",
+      "    Kube_CA_File     /var/run/secrets/kubernetes.io/serviceaccount/ca.crt",
+      "    Kube_Token_File  /var/run/secrets/kubernetes.io/serviceaccount/token",
+      "    Kube_Tag_Prefix  kube.var.log.containers.",
+      "    Merge_Log        Off",
+      "    Merge_Log_Key    log_processed",
+      "    Annotations      Off",
+      // Tweak the k8s labels to only keep the ones we care about
+      "[FILTER]",
+      "    Name    lua",
+      "    Match   kube.*",
+      "    script  /fluent-bit/scripts/k8s_labels.lua",
+      "    call    tweak_k8s_labels",
+    ].join("\n"),
+    customParsers: [
```

Comment:
Random side note: I think having our own instance also allows us fixing the parsing of e.g. the gha runners where everything gets parsed as a warning even if they don't add json support. Whether we care enough to do so is another question.